draft-ietf-quic-qpack.md

```diff
@@ -375,8 +375,9 @@ Insert Count. When the Required Insert Count is less than or equal to the
 decoder's Insert Count, the field section can be processed immediately.
 Otherwise, the stream on which the field section was received becomes blocked.
 
-While blocked, encoded field section data SHOULD remain in the blocked stream's
-flow control window.  A stream becomes unblocked when the Insert Count becomes
+To bound the memory requirements of the decoder for blocked streams, encoded
+field section data SHOULD remain in the stream's flow control window while the
+stream is blocked.  A stream becomes unblocked when the Insert Count becomes
```

Comment:
This seems to lack the "why" still.  Instructions that are read from the stream will be unusable while blocked but they need to be retained in memory AND the protocol does not provide any way to limit allocations for that memory.